### ISSUE TITLE
RSS feeds

### ISSUE THREAD

**Issue by Victor239** [legiblenews/community#7]
It would be helpful to be able to get news updates via RSS rather than manually checking the site.

**Comment by beaugunderson** [comment]
until Brad adds RSS feeds to Legible you can get an RSS of the wikipedia portal page that's not quite as pretty but has the same links and general layout here:

https://www.to-rss.xyz/wikipedia/current_events/

**Comment by bradgessler** [comment]
This is a great reminder that I need to get off my butt and create these feeds! I’m going to start work on this next.

**Comment by bradgessler** [comment]
K RSS feeds for daily news should be working via the auto discovery link from the daily issues. Could somebody try it out and LMK if its working as you'd expect? Feedback will be much appreciated.

I will likely make feeds a premium feature, but for now I'm making this feed available so that folks can have it now without waiting for me to do the work it takes to make a premium RSS feed.

Additionally, the feed URL will likely change as I clean up some of the code and architecture for this thing. I should be able to redirect the current URL to wherever the new URL is without breaking anything, but there's a non-zero chance that it breaks existing RSS feeds and they'd have to point to a new URL.

**Comment by beaugunderson** [comment]
very cool, just signed up for a paid account in support! 🙌

**Comment by bradgessler** [comment]
Thanks so much!

**Comment by bradgessler** [comment]
Just tried this in Feedly and it's not super great. What feed readers are people using these days that I should look at to test this whole thing out?

**Comment by beaugunderson** [comment]
I self-host a copy of [Miniflux](https://miniflux.app/) (with the "fetch original" option turned on if needed), I then point Reeder at the the Miniflux API. You could just add feeds directly to Reeder though, no need for anything fancy. :)

**Comment by bradgessler** [comment]
So looks like you click on the link and it takes you to the article? Feedly only displays a link; not the contents. I'll just leave this open from comments from others if they have opinions on making the RSS reading experience better.

For those who are joining, please be as specific as possible about the feed reader you're using and include screenshots. Thanks future commenters!

**Comment by beaugunderson** [comment]
Yeah your current feed doesn't include the article contents (compare the generated XML to the one [to-rss](https://www.to-rss.xyz/wikipedia/current_events/) makes)

Not a huge issue for e.g. miniflux since it's just a checkbox to make it fetch the original content automatically but most feed readers will just show the title

**Comment by gskoljarev** [comment]
Hi @bradgessler, long time reader, first time caller.

I've created a custom feed using [Feed43](https://feed43.com): http://feed43.com/4758275445775183.xml. I'm using https://www.inoreader.com to catch up with things and wanted to include Legible News in.

One thing I noticed is that the "...issue publishes..." and "Subscribe..." copy look as part of the news item. Perhaps moving to `footer` or a separate `section` might be a good idea. Feel free to ping me if you need to bounce a few ideas off of someone.

System: Windows 11, Firefox 105.0.3 (64-bit)

![Screenshot 2022-10-19 162328](https://user-images.githubusercontent.com/10615524/196719173-9b418b93-fa76-4554-abf7-5d7d344252a3.png)
![Screenshot 2022-10-19 162405](https://user-images.githubusercontent.com/10615524/196719177-e26636ad-08d4-4ab8-8d43-7bc056664eb3.png)

**Comment by bradgessler** [comment]
I'll probably replace that with a rotating message about Legible News Plus features, which won't be visible on Plus RSS feeds.

**Comment by bradgessler** [comment]
I don't have any specific timelines on when I'd do that, but it will probably be soonish since I'm about to change some things with the layout for an iOS app.

**Comment by edmundmiller** [comment]
Any updates on where the paid RSS feeds are? Not a game breaker since I'm using https://omnivore.app/ I could just use the newsletters.

**Comment by bradgessler** [comment]
No updates to report on RSS
